NeuroID/TrackerEvents/AppEvents.swift

```diff
@@ -30,4 +33,27 @@ internal extension NeuroIDTracker {
     @objc func appMovedToForeground() {
         captureEvent(event: NIDEvent(type: NIDEventName.windowFocus))
     }
+    
+    @objc func appLowMemoryWarning() {
+        // Reduce memory footprint
+        // Only clear this event queue the first time as it might be triggered a few times in a row (dropping our low mem event)
+        if (!NeuroID.lowMemory) {
+            DataStore.events = []
+            DataStore.queuedEvents = []
+            let lowMemEvent = NIDEvent(type: NIDEventName.lowMemory)
+            captureEvent(event: lowMemEvent)
```

Comment:
Do we still want to capture the low memory event if we are immediately saving it?